openfe/setup/alchemical_network_planner/relative_alchemical_network_planner.py

```diff
@@ -315,6 +315,50 @@ def __init__(
             protocol=protocol,
         )
 
+    def _build_transformation(
+        self,
+        ligand_mapping_edge: LigandAtomMapping,
+        stateA: ChemicalSystem,
+        stateB: ChemicalSystem,
+        transformation_protocol: Protocol,
+    ) -> Transformation:
+        """
+        Overwrite the default method to handle net charge change transformations with our default protocol.
+        """
+        transformation_name = self.name + "_" + stateA.name + "_" + stateB.name
+
+        # Todo: Another dirty hack! - START
```

Comment:
what needs to be done to address this? can you open an issue?

Reply:
Ah this was in the other transformation function I [copied](https://github.com/OpenFreeEnergy/openfe/blob/d2f47194fb955c15861809d89cd456ad383202a1/openfe/setup/alchemical_network_planner/relative_alchemical_network_planner.py#L213-L216) I'll remove it.